docs/docs/92-awesome.md

```diff
@@ -25,9 +25,15 @@ If you have some missing resources, please feel free to [open a pull-request](ht
 
 ## Templates
 
-## Blogs, guides, videos
+## Posts & tutorials
 
-- [Tutorial about setup of Woodpecker-CI and Gitea](https://containers.fan/posts/setup-gitea-with-woodpecker-ci/)
+- [Setup Gitea with Woodpecker CI](https://containers.fan/posts/setup-gitea-with-woodpecker-ci/)
```

Comment:
oh nice where did you got that list from :)

Reply:
google :wink: